lib/intercom/index.js

```diff
@@ -86,29 +86,31 @@ Intercom.prototype.identify = function(identify){
 
   traits.app_id = this.options.appId;
 
-  // intercom requires `company` to be an object. default it with group traits
-  // so that we guarantee an `id` is there, since they require it
-  if (null != traits.company && !is.object(traits.company)) delete traits.company;
-  if (traits.company) defaults(traits.company, group.traits());
-
   // name
   if (name) traits.name = name;
 
   // handle dates
-  if (created) {
+  if (identify.created()) {
```

Comment:
Any reason we're calling this twice rather than caching it in a variable like before?